### DISCUSSION TITLE
Combobox event types

### DISCUSSION THREAD

**Discussion by hafffe** [chakra-ui/ark#2046] [Q&A] [answered]
Hello,
I'm trying to find how I can access the events for `onValueChange` and `onInputValueChange`.
As you can see here https://github.com/chakra-ui/ark/blob/main/packages/website/src/demos/combobox.demo.tsx#L16 `e` is set to `any` so we have no idea what the `e` parameter is.

Is there any method to make `e` in the example correctly typed?

**Comment by Omikorin** [comment, marked as answer]
Hi, you can import `ComboboxValueChangeDetails`.
Docs are a work in progress (as always).

Feel free to contribute by revising missing typings in stories, docs and demos :)

**Reply by hafffe** [reply]
Somehow I missed that this was exported from ark.
Thanks!
